packages/account/src/Components/form-sub-header/form-sub-header.tsx

```diff
@@ -1,34 +1,38 @@
-import classNames from 'classnames';
 import React from 'react';
 import { Text } from '@deriv/components';
+import classNames from 'classnames';
 
 export type TFormSubHeader = {
     description?: string;
     subtitle?: string;
-    title: string;
+    title?: string;
 };
 
 export const FormSubHeader = ({ description, subtitle, title }: TFormSubHeader) => {
-    const title_as_class = title.replace(/\s+/g, '-').toLowerCase();
+    const title_as_class = title?.replace(/\s+/g, '-').toLowerCase();
     return (
         <React.Fragment>
-            <div
-                className={classNames('account-form__header', title_as_class, {
-                    'account-form__header--has-description': !!description,
-                })}
-                data-testid='form-sub-header'
-            >
-                <div className='account-form__header-section'>
-                    <Text as='h1' color='prominent' weight='bold' size='xs' className='account-form__title'>
-                        {title}
-                    </Text>
-                    {subtitle && (
-                        <Text as='h2' size='xxxs' color='prominent' className='account-form__subtitle'>
-                            {subtitle}
-                        </Text>
-                    )}
+            {(title || subtitle) && (
```

Comment:
This FormSubHeader is used in multiple places. Can we add a checklist to them to make sure style is not broken?